### ISSUE TITLE
Translators Needed

### ISSUE THREAD

**Issue by theweavrs** [theweavrs/BreadPlayer#144]
If anyone would like to translate BreadPlayer into their native language, they are more than welcome. Please comment below with the language you are going to translate so I can add it to the list of WIP languages and avoid causing conflicts. 

Please follow the guide here on [how to translate BreadPlayer.](https://github.com/theweavrs/BreadPlayer/wiki/I-Want-To-Translate!)

### Translations Done:
1. English US (en-US)

### Translation WIP:
1. Czech (cs-CZ)

**Comment by ududsha** [comment]
Is there any chance to translate it into Sinhala?

**Comment by theweavrs** [comment]
@ududsha of course. If you want to go for it then I will reserve it for you.

**Comment by ududsha** [comment]
@theweavrs please reserve it for me. Thanks!

**Comment by theweavrs** [comment]
@ududsha done. Do you know how to go about the process? Have you done translation before for any other project?

**Comment by ududsha** [comment]
@theweavrs this is my first time in a translation job. I have gone through the guide. I'll fork the codebase and pull it for review.

**Comment by theweavrs** [comment]
Thanks. If you need any help or guidance, we can chat on Gitter. [Here's the room link.](https://gitter.im/BreadPlayer/Lobby)

**Comment by ududsha** [comment]
Now we've done with Sinhala. Cheers!

**Comment by HolzingerAlexander** [comment]
I would like to do the translation to german.

**Comment by theweavrs** [comment]
@HolzingerAlexander de-DE will work wonderfully. Thanks.

**Comment by cdytoby** [comment]
I would like to translate it into Chinese (it should be zh-cn). Seems not a complicate work. But I have a question to ask: do you really translate "bread" or "toast" into "bread" or "toast" instead of songs? It would be confusing, since it's an audio player instead of "toaster".

**Comment by theweavrs** [comment]
@cdytoby it is what it is. Sure, I'd really alprecoate it if you do a Chinese translation. And yes, "bread" and "toasts" are both translated into native language. This is Bread's own terminology. A metaphor you can say.

**Comment by SoColdWinter** [comment]
Hello @theweavrs , I would like to translate your app to spanish because it seems interesting and spanish is my first language :P.
It is my first contribution because i never did it, may I ask you about a guide?
Thank you in advance!

**Comment by CharlieFoxTwo** [comment]
I'm looking to get to work on a Russian translation in a bit. I've noticed some misspellings in the English  version ("Appearence" should be "Appearance"...), would you like me to do anything about those?

**Comment by theweavrs** [comment]
@CharlieFoxTwo Yes, there may be a few misspellings here and there. Correct them as you go and later on send me a list of the corrected words if you can. I will also look into it myself. Thanks.

**Comment by mickaj** [comment]
Hi there! I'll happily do Polish (pl-PL). Cheers!

**Comment by theweavrs** [comment]
@mickaj I will add your name into the list! Welcome to Bread Family 😆

**Comment by theweavrs** [comment]
@charliefoxtwo can you start the work on Russian Translation? Would really appreciate it.

**Comment by Natank7** [comment]
want translate to PT-BR

**Comment by theweavrs** [comment]
@Natank7 I added you name to the WIP list. Please proceed with the translation.

**Comment by Natank7** [comment]
Can i adapt some expressions to the 'theme'? Like: Library -> Pantry (in my language, of course)

**Comment by theweavrs** [comment]
@Natank7 As much as I would like to do that, it will be too confusing. People are already asking to use the normal theme instead of the Bread theme. So...just go with the flow.

**Comment by Natank7** [comment]
ok

**Comment by theweavrs** [comment]
@mickaj @ududsha @Natank7 In the last few versions, the Resource.resw file was changed, I would like you to re-translate the updated strings if you have time. Thanks. Please use [ResXManager ](https://clickonce-tom-englert.azurewebsites.net/ResXResourceManager/ResXManager.application)for faster translations.

**Comment by mickaj** [comment]
@theweavrs I'll take care of that during the weekend. can't do it any time sooner, sorry.

**Comment by theweavrs** [comment]
@mickaj No problem. As long as you can do it, all's well and good. Thanks.

**Comment by ahmedflix25** [comment]
Resources.resw the file is 404

**Comment by theweavrs** [comment]
@ahmedflix25 Fixed. Thanks for informing me :)

**Comment by ahmedflix25** [comment]
still a 404
https://github.com/theweavrs/BreadPlayer/wiki/I-Want-To-Translate!
https://raw.githubusercontent.com/theweavrs/BreadPlayer/master/BreadPlayer.Views.UWP/Strings/enww/Resources.resw

**Comment by theweavrs** [comment]
@ahmedflix25 check now! 😆 Seems I made a mistake again :)

**Comment by ahmedflix25** [comment]
well, now the whole wiki page is 404
going to the link just redirect me to the wiki home page which have nothing to do with the translation file anyway i pulled the whole source code and got it

**Comment by santosh** [comment]
Well, I want to make Hindi translation.

**Comment by Natank7** [comment]
Link to the RESW file
https://raw.githubusercontent.com/theweavrs/BreadPlayer/master/BreadPlayer.Views.UWP/Strings/en/Resources.resw

**Comment by 89netraM** [comment]
Hi! I'll make a Swedish (`sv-se`) translation.  
Will open a PR soon.

**Comment by TomerPacific** [comment]
Hi @thecodrr , I'd like to translate BreadPlayer to **Hebrew** (he-IL).
Let me know if this is ok.

**Comment by mai-am** [comment]
Hey, @thecodrr 
I'd be happy to make a Norwegian (nb-no) translation. Can do it next week! :D
